Kitodo-API/src/test/java/org/kitodo/utils/GuardTest.java

```diff
@@ -0,0 +1,184 @@
+        Object input = "Hello world!";
+        String greet = Guard.canCast("input", input, String.class);
+        assertTrue(greet instanceof String, "should return String");
+    }
+
+    @Test
+    @Tag("canCast")
+    void canCastShouldCastToSuperclass() {
+        Object input = new GregorianCalendar();
+        Calendar calendar = Guard.canCast("input", input, Calendar.class);
+        assertTrue(calendar instanceof Calendar, "should return Calendar");
+    }
+
+    @Test
+    @Tag("canCast")
+    void canCastShouldNotMiscast() {
+        IllegalArgumentException e = assertThrows(IllegalArgumentException.class, () -> {
+            Object input = new ArrayList<File>();
+            Guard.canCast("input", input, String.class);
+        }, "should throw IllegalArgumentException");
+        assertEquals("java.util.ArrayList 'input' is not a (subclass of) java.lang.String", e.getMessage(),
+            "should provide an understandable description");
+    }
+
+    @Test
+    @Tag("isInRange")
+    void isInRangeShouldFailBelowLowerBound() {
+        IllegalArgumentException e = assertThrows(IllegalArgumentException.class, () -> {
+            long dice = 0;
+            Guard.isInRange("dice", dice, 1, 6);
+        });
+        assertEquals("'dice' out of range: 0 not in [1..6]", e.getMessage(),
+            "should provide an understandable description");
+    }
+
+    @Test
+    @Tag("isInRange")
+    void isInRangeShouldNotFailOnLowerBound() {
+        long dice = 1;
+        Guard.isInRange("dice", dice, 1, 6);
+    }
+
+    @Test
+    @Tag("isInRange")
+    void isInRangeShouldNotFailInRange() {
+        long dice = 3;
+        Guard.isInRange("dice", dice, 1, 6);
+    }
+
+    @Test
+    @Tag("isInRange")
+    void isInRangeShouldNotFailOnUpperBound() {
+        long dice = 6;
+        Guard.isInRange("dice", dice, 1, 6);
+    }
+
+    @Test
+    @Tag("isInRange")
+    void isInRangeShouldFailAboveUpperBound() {
+        IllegalArgumentException e = assertThrows(IllegalArgumentException.class, () -> {
+            long dice = 7;
+            Guard.isInRange("dice", dice, 1, 6);
+        }, "should throw IllegalArgumentException");
+        assertEquals("'dice' out of range: 7 not in [1..6]", e.getMessage(),
+            "should provide an understandable description");
+    }
+
+    @Test
+    @Tag("isNotNull")
+    void isNotNullShouldNotFailForInitializedObjekt() {
+        Object object = "Hello world!";
+        Guard.isNotNull("object", object);
+    }
+
+    @Test
+    @Tag("isNotNull")
+    void isNotNullShouldFailForNullObjekt() {
+        IllegalArgumentException e = assertThrows(IllegalArgumentException.class, () -> {
+            Object object = null;
+            Guard.isNotNull("object", object);
+        }, "should throw IllegalArgumentException");
+        assertEquals("'object' must not be null", e.getMessage(), "should provide an understandable description");
+    }
+
+    @Test
+    @Tag("isPositive(double)")
```

Comment:
This  and seven more similar lines cause warnings:
```
Jul 05, 2024 6:24:52 PM org.junit.jupiter.engine.descriptor.JupiterTestDescriptor lambda$getTags$1
WARNING: Configuration error: invalid tag syntax in @Tag("isPositive(double)") declaration on [void org.kitodo.utils.GuardTest.isPositiveDoubleShouldFailForZero()]. Tag will be ignored.
```

Should these lines be removed?